lib/ood_core/job/adapters/lsf.rb

```diff
@@ -90,15 +90,18 @@ def submit(script, after: [], afterok: [], afternotok: [], afterany: [])
         # @return [Info] information describing submitted job
         # @see Adapter#info
         def info(id)
-          # TODO: handle job arrays
-          job = batch.get_job(id: id)
-          if job
-            info_for_batch_hash(job)
-          else
+          info_ary = batch.get_job(id: id).map{|v| info_for_batch_hash(v)}
+
+          case info_ary.size
+          when 0
             Info.new(
               id: id,
               status: :completed
             )
+          when 1
+            info_ary.first
+          else  # many
+            handle_job_array(info_ary, id)
           end
```

Comment:
It might be more readable to just handle the edge cases at the top of handle_job_array:

```ruby
def info(id)
  info_ary = batch.get_job(id: id).map{|v| info_for_batch_hash(v)}
  handle_job_array(info_ary, id)
rescue Batch::Error => e
  # ...
end

def handle_job_array(info_ary, id)
  return Info.new(id: id, status: :completed) if info_ary.nil? || info_ary.empty?
  return info_ary.first if info_ary.size == 1

  parent_task_hash = build_proxy_parent(info_ary.first, id)
  #...
```